tailwind.config.js

```diff
@@ -44,6 +44,9 @@ module.exports = {
           },
         },
       },  
+      height: {
+        "calc-full-0.75": "calc(100% - 0.75rem)",
```

Comment:
Lets use single quotes for strings in all Javascript code. The exception would be properties or attributes in JSX (`className="foo"`).

Reply:
Sure!